src/components/EncounterNotes/SabethaNotes.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+
+import RoleEmbed from "./../RoleEmbed/RoleEmbed";
+
+function SabethaNotes() {
+  return (
+    <div>
+      <h3>General</h3>
+      <ul>
+        <li>
+          <RoleEmbed role="Heal" profession="druid" /> kites.
+        </li>
+      </ul>
+      <h3>Cannons</h3>
```

Comment:
For Cannon's we usually have BS on 2-4 lately, now that Miruna is doing them.. doesn't really matter which does what I suppose but if we want it to be accurate

Reply:
Ah mb, tbh I mostly copy pasted content from the spreadsheet, didn't really analyze it thoroughly

Reply:
Done